Sources/Slacket/Parsers/ParserType.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ParserType.swift
+//  Slacket
+//
+//  Created by Jakub Tomanik on 20/05/16.
+//
+//
+
+import Foundation
+import Kitura
+
+// To define models that can be parsed
+protocol Parsable {}
+
+// To define parsers
+protocol ParserType {}
+
+protocol ParserDecoder: ParserType {
+    
+    func parse(body: ParsedBody) -> Parsable
```

Comment:
Heh, sorry, silly mistake. What I'd like to say is that we'd like to return `SlackComand` from `parse` method in `SlackCommandDecoder`, but now we return some `Parsable` object.